alembic/versions/e2e504d6fa14_change_unique_of_authors_name.py

```diff
@@ -0,0 +1,30 @@
+"""change unique of authors name
+
+Revision ID: e2e504d6fa14
+Revises: e00855069317
+Create Date: 2024-10-28 17:33:36.062044
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'e2e504d6fa14'
+down_revision: Union[str, None] = 'e00855069317'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'author', ['name'])
```

Comment:
The unique constraint is being created with `None` as its name. It's advisable to provide a specific name for the constraint to avoid potential issues when referencing it in the future. You can do this by replacing `None` with a string, e.g., `'uq_author_name'`.